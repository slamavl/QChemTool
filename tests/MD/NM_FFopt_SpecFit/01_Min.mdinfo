

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   7175       1.9993E+03     9.5430E-05     3.3731E-04     C35        48

 BOND    =      162.4367  ANGLE   =     1112.0934  DIHED      =      372.7289
 VDWAALS =      166.8612  EEL     =        3.2000  HBOND      =        0.0000
 1-4 VDW =      189.4034  1-4 EEL =       -7.4719  RESTRAINT  =        0.0000
