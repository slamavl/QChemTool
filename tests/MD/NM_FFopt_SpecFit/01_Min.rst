FGR 
  262
   0.0699576   9.8706131   6.4567768  -0.6568082  10.5394976   5.1644478
  -0.0156357   9.7452162   3.8862831  -0.3684733   8.2348281   3.8566210
   0.0975537   7.5163803   5.1488555  -0.2663391   6.0053692   5.1784647
   0.2138604   5.3274136   6.4895160  -0.2059718   6.1393189   7.8421359
   0.2228838   7.7014043   7.7787432   1.5681581   7.8225452   7.8382233
  -0.4532171   8.2272141   8.8244889  -1.5354218   6.0306412   8.0491246
   0.5996290   5.7692482   8.8617150  -0.2332196   3.8043296   6.4851544
   0.2698833   3.0683085   7.8395447  -0.2000875   1.5255779   7.8637243
   0.2471446   0.7520797   6.5112474  -0.1908854   1.4778931   5.2176858
   0.1880033   3.0455284   5.1877080  -0.1395153   3.6695655   3.7719321
   0.1550413   5.2215248   3.8947564  -0.2596344   5.9867709   2.5316995
   0.1096466   7.5032496   2.5705097  -0.3248138   8.2556248   1.2835857
   0.0514778   9.7566653   1.2942062  -0.5447264  10.5603567   2.5811735
  -1.8926007  10.5718955   2.5845759   0.0486360  11.7718810   2.5734491
  -0.4254595  10.5813693  -0.0248426   0.1705029   9.7551453  -1.2972007
  -0.2369023   8.2620249  -1.3065632   0.1671976   7.5381615   0.0026234
  -0.2348931   6.0437980  -0.0068699   0.0887842   5.1178098   1.2531818
   0.0073500   3.5880708   1.2362581  -0.0264204   2.8942663   2.4661969
  -0.0044783   1.4801398   2.4815319   0.1241028   0.7748400   3.8423517
  -0.1241126  -0.7748485   3.8423526   0.1908901  -1.4778943   5.2176866
  -0.2471355  -0.7520807   6.5112494   0.2001018  -1.5255781   7.8637251
  -0.2698681  -3.0683091   7.8395482   0.2332326  -3.8043295   6.4851569
  -0.1879940  -3.0455291   5.1877109   0.1395281  -3.6695648   3.7719353
   0.0264332  -2.8942792   2.4661914   0.0044531  -1.4801508   2.4815312
   0.0053134  -0.7188372   1.2570124  -0.0053871   0.7188296   1.2570138
  -0.0116548   1.4357260  -0.0004693   0.0089783   2.8835807  -0.0008005
   0.0179252   3.5877608  -1.2380872   0.1237448   5.1159544  -1.2528015
  -0.1589856   5.9938457  -2.5442384   0.2555680   7.4982919  -2.5641072
  -0.1144925   8.2491799  -3.8773387   0.3163072   9.7360629  -3.8831371
  -0.2730286  10.5833934  -2.6243637  -1.6178548  10.6563151  -2.6736527
   0.3786572  11.7643986  -2.6189241  -0.1032033  10.5963816  -5.1977585
   0.5355529   9.8369103  -6.4796484   0.0472676   8.2856925  -6.4181413
   0.3864899   7.5005315  -5.1315292  -0.0631517   6.0088631  -5.1870269
   0.2828993   5.2116621  -3.8881534  -0.0622307   3.6715017  -3.7759422
  -0.0083428   2.8940423  -2.4680491   0.0112984   1.4797194  -2.4825352
  -0.0018614   0.7188391  -1.2577570   0.0016206  -0.7188402  -1.2577643
   0.0115319  -1.4357274  -0.0004750  -0.0090408  -2.8835819  -0.0008035
  -0.0073051  -3.5880767   1.2362489  -0.0887691  -5.1178134   1.2531862
   0.2596447  -5.9867725   2.5317057  -0.1550287  -5.2215245   3.8947624
   0.2663532  -6.0053699   5.1784692  -0.2138459  -5.3274140   6.4895204
   0.2059880  -6.1393197   7.8421395  -0.2228683  -7.7014049   7.7787469
   0.3745145  -8.2983202   6.3967693  -0.0975392  -7.5163810   5.1488591
   0.3684870  -8.2348294   3.8566247  -0.1096347  -7.5032514   2.5705138
   0.3248255  -8.2556271   1.2835898  -0.1671902  -7.5381675   0.0026273
   0.2348954  -6.0438033  -0.0068663  -0.1237614  -5.1159638  -1.2527970
  -0.0179836  -3.5877675  -1.2380840   0.0081862  -2.8940496  -2.4680441
  -0.0116976  -1.4797339  -2.4825402  -0.1382187  -0.7706929  -3.8418890
   0.1379994   0.7707156  -3.8418831  -0.1550828   1.4812541  -5.2181270
   0.2937940   3.0338650  -5.1786389  -0.0533752   3.8001208  -6.4925014
   0.4320997   5.3107616  -6.4831016   0.0892937   6.1595725  -7.8398945
   0.7565729   7.6350483  -7.7292695   2.1032242   7.5559779  -7.6346594
   0.3064944   8.2685899  -8.8339532  -1.2353393   6.1802463  -8.0919810
   0.8808830   5.7338177  -8.8472215   1.8963540   5.2299544  -6.4339294
   0.4808644   3.0256214  -7.8154153  -0.1863870   1.5549546  -7.8531394
   0.2569638   0.7486821  -6.5157071  -0.2569962  -0.7486739  -6.5157151
   0.1549911  -1.4812331  -5.2181083  -0.2938500  -3.0338575  -5.1786385
   0.0621443  -3.6714997  -3.7759354  -0.2829277  -5.2116697  -3.8881518
   0.1589745  -5.9938483  -2.5442366  -0.2555621  -7.4982991  -2.5641062
   0.2369124  -8.2620274  -1.3065607  -0.1704830  -9.7551504  -1.2971999
   0.4254798 -10.5813709  -0.0248396  -0.0514623  -9.7566685   1.2942088
   0.5447409 -10.5603587   2.5811772   0.0156492  -9.7452174   3.8862860
   0.6568199 -10.5394992   5.1644513  -0.0699462  -9.8706137   6.4567795
  -1.4060787 -10.0535230   6.4317025   0.5521577 -10.4651471   7.4964201
   1.9993881 -10.4387291   5.2500248   0.1929794 -11.8062717   5.1704770
  -1.4381441  -9.9511619   3.9174731   1.8926152 -10.5718974   2.5845805
  -0.0486217 -11.7718829   2.5734524  -1.5115560  -9.9000447   1.3636414
   1.7721881 -10.6252942  -0.0487287  -0.1984166 -11.7771738  -0.0247127
   0.2730591 -10.5833938  -2.6243626  -0.3162758  -9.7360672  -3.8831393
   0.1145110  -8.2491806  -3.8773377  -0.3864696  -7.5005345  -5.1315309
   0.0631538  -6.0088603  -5.1870225  -0.4320854  -5.3107581  -6.4831007
   0.0533749  -3.8001113  -6.4924866  -0.4808262  -3.0256208  -7.8154202
   0.1864176  -1.5549500  -7.8531233   1.5295979  -1.5766852  -7.9999807
  -0.5383684  -1.1039618  -8.9009772  -1.8307103  -2.9967315  -7.8770922
   0.1721569  -3.5036099  -8.8966905   1.5142470  -3.8432540  -6.6218980
  -0.0892526  -6.1595634  -7.8398897  -0.7565208  -7.6350456  -7.7292764
  -0.0472264  -8.2856883  -6.4181417  -0.5354995  -9.8369098  -6.4796545
   0.1032507 -10.5963793  -5.1977601   1.4440912 -10.6929310  -5.3208125
  -0.5672404 -11.7688492  -5.2080256  -1.8819339  -9.9370934  -6.4696906
   0.0843293 -10.4634759  -7.5040564   1.4040167  -8.3266063  -6.6443851
  -2.1031738  -7.5559877  -7.6346818  -0.3064238  -8.2685809  -8.8339561
   1.2353846  -6.1802227  -8.0919558  -0.8808310  -5.7338109  -8.8472268
  -1.8963412  -5.2299680  -6.4339501   1.5255048  -6.0384400  -5.2853345
  -1.8512082  -7.5095690  -5.0648568   1.5774610  -8.2939168  -3.9705773
  -1.7806252  -9.8215378  -3.7997826   1.6178858 -10.6563066  -2.6736458
  -0.3786185 -11.7644036  -2.6189269  -1.6322235  -9.8713963  -1.2093445
   1.7026654  -8.2725966  -1.3573101  -1.7201778  -7.5079143  -2.5002551
   1.6226425  -5.9733875  -2.6173300  -1.7458098  -5.1790234  -3.8043199
   1.5175943  -3.7421746  -3.9061168  -1.7561106  -2.9487729  -5.1278667
   1.6189623  -1.5462528  -5.2406076  -1.7228233  -0.6902735  -6.5512746
   1.7227925   0.6902796  -6.5511929  -1.5295609   1.5766997  -8.0000546
   0.5384393   1.1039569  -8.9009611   1.8307504   2.9967195  -7.8770442
  -0.1720784   3.5036110  -8.8967095  -1.5142418   3.8432867  -6.6219662
   1.7560514   2.9487514  -5.1278242  -1.6190517   1.5463044  -5.2407172
   1.5983856   0.6667316  -3.8547743  -1.5986018  -0.6666912  -3.8549142
  -1.5851922  -5.1972975  -1.2358995   1.6965059  -5.9304902  -0.0404161
  -1.6324645  -7.5512883   0.0536838   1.7909652  -8.2364211   1.2476567
  -1.5739033  -7.5624625   2.6108091   1.8343705  -8.1884644   3.8563703
  -1.5608469  -7.6059586   5.1869288   1.8350137  -8.2322227   6.5467118
  -1.5681426  -7.8225454   7.8382278   0.4532332  -8.2272152   8.8244919
   1.5354383  -6.0306419   8.0491260  -0.5996112  -5.7692489   8.8617197
  -1.6817550  -5.3392742   6.5008214   1.7307287  -5.9601878   5.2205173
  -1.6202969  -5.2466445   3.8758225   1.7233796  -5.9113616   2.5381645
  -1.5478135  -5.2232697   1.3057981  -1.5176746   3.7422118  -3.9061743
   1.7457802   5.1789908  -3.8043108  -1.5255010   6.0384609  -5.2853596
   1.8512279   7.5095487  -5.0648419  -1.4039728   8.3266229  -6.6443998
   1.8819880   9.9370840  -6.4696729  -0.0842628  10.4634829  -7.5040542
  -1.4440423  10.6929426  -5.3208206   0.5672960  11.7688468  -5.2080180
   1.7806568   9.8215198  -3.7997715  -1.5774418   8.2939277  -3.9705861
   1.7201834   7.5078894  -2.5002505  -1.6226534   5.9734002  -2.6173333
   1.5851773   5.1972605  -1.2359037   1.6001637  -3.6849824   3.8568299
  -1.6531658  -3.0205447   5.1873455   1.6981682  -3.7711587   6.5671205
  -1.6157314  -3.1396076   7.9431868   0.4515321  -3.4987502   8.8976866
   1.5413113  -1.4517606   8.0154550  -0.5611773  -1.1150743   8.9030574
  -1.7146152  -0.7549656   6.5450939   1.6560556  -1.4896317   5.2355301
  -1.5869095  -0.7198966   3.8660096   1.5868999   0.7198911   3.8659955
   1.5478275   5.2232799   1.3057912  -1.6965039   5.9304910  -0.0404146
   1.6324718   7.5512769   0.0536794  -1.7026554   8.2726032  -1.3573137
   1.6322436   9.8713818  -1.2093408  -1.7721677  10.6252983  -0.0487341
   0.1984419  11.7771697  -0.0247142   1.5115715   9.9000384   1.3636414
  -1.7909537   8.2364225   1.2476537   1.5739151   7.5624613   2.6108057
  -1.7233692   5.9113603   2.5381608   1.6203095   5.2466432   3.8758162
  -1.6001509   3.6849837   3.8568266   1.6531750   3.0205407   5.1873392
  -1.6560508   1.4896341   5.2355363   1.7146246   0.7549647   6.5450858
  -1.5412964   1.4517613   8.0154587   0.5611949   1.1150740   8.9030541
   1.6157469   3.1396060   7.9431800  -0.4515139   3.4987502   8.8976850
  -1.6981551   3.7711605   6.5671210   1.6817694   5.3392724   6.5008154
  -1.7307145   5.9601872   5.2205140   1.5608613   7.6059582   5.1869250
  -1.8343567   8.1884630   3.8563672   1.4381576   9.9511618   3.9174721
  -1.9993762  10.4387259   5.2500204  -0.1929692  11.8062707   5.1704735
   1.4060898  10.0535247   6.4317016  -0.5521489  10.4651453   7.4964166
  -0.3745004   8.2983190   6.3967660  -1.8349993   8.2322200   6.5467090
