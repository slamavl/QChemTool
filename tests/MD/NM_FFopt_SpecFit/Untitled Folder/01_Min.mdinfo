

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1511       1.8420E+03     9.6606E-05     3.8660E-04     C138      305

 BOND    =      205.8039  ANGLE   =      491.5732  DIHED      =      672.9939
 VDWAALS =      291.3301  EEL     =        5.1582  HBOND      =        0.0000
 1-4 VDW =      231.1369  1-4 EEL =      -55.9856  RESTRAINT  =        0.0000
