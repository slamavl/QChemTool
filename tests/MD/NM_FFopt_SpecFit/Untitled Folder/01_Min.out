
          -------------------------------------------------------
          Amber 14 SANDER                              2014
          -------------------------------------------------------

| Run on 06/17/2018 at 14:36:19

|   Executable path: sander
| Working directory: /mnt/sda2/Dropbox/PhD/Programy/Python/QChemTool/tests/MD/NM_FFopt_SpecFit
|          Hostname: Unknown
  [-O]verwriting output

File Assignments:
|  MDIN: 01_Min.in                                                             
| MDOUT: 01_Min.out                                                            
|INPCRD: nab_input.inpcrd                                                      
|  PARM: nab_input.prmtop                                                      
|RESTRT: 01_Min.rst                                                            
|  REFC: refc                                                                  
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: mden                                                                  
| MDCRD: 01_Min.mdcrd                                                          
|MDINFO: 01_Min.mdinfo                                                         
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

minimize                                                                       
 &cntrl                                                                        
  imin=1,                                                                      
  ntb=0,                                                                       
  ntx=1,                                                                       
  irest=0,                                                                     
  maxcyc=10000,                                                                
  ncyc=4000,                                                                   
  ntpr=100,                                                                    
  ntwx=10,                                                                     
  cut=999.0,                                                                   
 /                                                                             

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
|    NONPERIODIC   ntb=0 and igb=0: Setting up nonperiodic simulation
|  *** cutoff > system size, list only builds once
|Largest sphere to fit in unit cell has radius =    31.728
| New format PARM file being parsed.
| Version =    1.000 Date = 06/17/18 Time = 14:36:19
 NATOM  =     330 NTYPES =       2 NBONH =       0 MBONA  =     390
 NTHETH =       0 MTHETA =     900 NPHIH =       0 MPHIA  =    3126
 NHPARM =       0 NPARM  =       0 NNB   =    2998 NRES   =       1
 NBONA  =     390 NTHETA =     900 NPHIA =    3126 NUMBND =       5
 NUMANG =      10 NPTRA  =       8 NATYP =       4 NPHB   =       0
 IFBOX  =       0 NMXRS  =     330 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0


|     Memory Use     Allocated
|     Real               30567
|     Hollerith            993
|     Integer           174685
|     Max Pairs          54285
|     nblistReal          3960
|     nblist Int       1528485
|       Total             7138 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

FGR                                                                             

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     100, ntrx    =       1, ntwr    =     500
     iwrap   =       0, ntwx    =      10, ntwv    =       0, ntwe    =       0
     ioutfm  =       0, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       0, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       0

Energy minimization:
     maxcyc  =   10000, ncyc    =    4000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010
|  INFO: Old style inpcrd file read


--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

FGR                                                                             
 begin time read from input coords =     0.000 ps

 Number of triangulated 3-point waters found:        0

     Sum of charges from parm topology file =   0.00000000
     Forcing neutrality...

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

 ---------------------------------------------------

     eedmeth=4: Setting switch to one everywhere

 ---------------------------------------------------
| Local SIZE OF NONBOND LIST =      51288
| TOTAL SIZE OF NONBOND LIST =      51288


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1       2.0671E+03     2.6049E+01     1.0193E+02     C7         19

 BOND    =      349.2878  ANGLE   =      565.5696  DIHED      =      686.2505
 VDWAALS =      290.9881  EEL     =        5.5188  HBOND      =        0.0000
 1-4 VDW =      226.0755  1-4 EEL =      -56.5537  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    100       1.8421E+03     6.0958E-02     3.1097E-01     C150        4

 BOND    =      205.2363  ANGLE   =      490.9739  DIHED      =      673.6187
 VDWAALS =      291.8462  EEL     =        5.1730  HBOND      =        0.0000
 1-4 VDW =      231.2779  1-4 EEL =      -56.0074  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    200       1.8420E+03     3.7099E-02     1.5711E-01     C109      242

 BOND    =      205.6765  ANGLE   =      491.4643  DIHED      =      673.1365
 VDWAALS =      291.3377  EEL     =        5.1595  HBOND      =        0.0000
 1-4 VDW =      231.2308  1-4 EEL =      -55.9877  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    300       1.8420E+03     1.1471E-02     5.1907E-02     C112      253

 BOND    =      205.7449  ANGLE   =      491.5948  DIHED      =      673.0466
 VDWAALS =      291.2753  EEL     =        5.1578  HBOND      =        0.0000
 1-4 VDW =      231.1782  1-4 EEL =      -55.9851  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    400       1.8420E+03     2.2840E-03     9.2576E-03     F169       21

 BOND    =      205.7836  ANGLE   =      491.5932  DIHED      =      673.0203
 VDWAALS =      291.2781  EEL     =        5.1578  HBOND      =        0.0000
 1-4 VDW =      231.1637  1-4 EEL =      -55.9851  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500       1.8420E+03     2.6878E-03     8.2202E-03     C109      242

 BOND    =      205.7938  ANGLE   =      491.5893  DIHED      =      673.0104
 VDWAALS =      291.2889  EEL     =        5.1579  HBOND      =        0.0000
 1-4 VDW =      231.1560  1-4 EEL =      -55.9852  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    600       1.8420E+03     3.1607E-03     1.2533E-02     C5         14

 BOND    =      205.7921  ANGLE   =      491.5915  DIHED      =      673.0056
 VDWAALS =      291.2989  EEL     =        5.1579  HBOND      =        0.0000
 1-4 VDW =      231.1501  1-4 EEL =      -55.9852  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    700       1.8420E+03     2.5112E-03     7.9914E-03     C109      242

 BOND    =      205.7977  ANGLE   =      491.5843  DIHED      =      673.0023
 VDWAALS =      291.3069  EEL     =        5.1580  HBOND      =        0.0000
 1-4 VDW =      231.1469  1-4 EEL =      -55.9854  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    800       1.8420E+03     6.9474E-04     2.8111E-03     F148       62

 BOND    =      205.7997  ANGLE   =      491.5810  DIHED      =      672.9999
 VDWAALS =      291.3131  EEL     =        5.1581  HBOND      =        0.0000
 1-4 VDW =      231.1443  1-4 EEL =      -55.9854  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    900       1.8420E+03     2.1636E-03     1.0798E-02     C11        30

 BOND    =      205.8043  ANGLE   =      491.5750  DIHED      =      672.9981
 VDWAALS =      291.3179  EEL     =        5.1581  HBOND      =        0.0000
 1-4 VDW =      231.1428  1-4 EEL =      -55.9855  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000       1.8420E+03     3.9702E-04     1.6624E-03     F148       62

 BOND    =      205.8019  ANGLE   =      491.5770  DIHED      =      672.9970
 VDWAALS =      291.3214  EEL     =        5.1581  HBOND      =        0.0000
 1-4 VDW =      231.1407  1-4 EEL =      -55.9855  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1100       1.8420E+03     5.1193E-04     1.9684E-03     C138      305

 BOND    =      205.8022  ANGLE   =      491.5762  DIHED      =      672.9960
 VDWAALS =      291.3242  EEL     =        5.1581  HBOND      =        0.0000
 1-4 VDW =      231.1394  1-4 EEL =      -55.9856  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1200       1.8420E+03     3.5917E-04     1.1197E-03     F28       306

 BOND    =      205.8031  ANGLE   =      491.5747  DIHED      =      672.9952
 VDWAALS =      291.3263  EEL     =        5.1582  HBOND      =        0.0000
 1-4 VDW =      231.1386  1-4 EEL =      -55.9856  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1300       1.8420E+03     4.3786E-04     1.7967E-03     C5         14

 BOND    =      205.8029  ANGLE   =      491.5747  DIHED      =      672.9947
 VDWAALS =      291.3279  EEL     =        5.1582  HBOND      =        0.0000
 1-4 VDW =      231.1378  1-4 EEL =      -55.9856  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1400       1.8420E+03     4.0052E-04     1.4314E-03     C11        30

 BOND    =      205.8040  ANGLE   =      491.5732  DIHED      =      672.9943
 VDWAALS =      291.3291  EEL     =        5.1582  HBOND      =        0.0000
 1-4 VDW =      231.1374  1-4 EEL =      -55.9856  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500       1.8420E+03     2.9974E-04     1.2446E-03     C73       169

 BOND    =      205.8040  ANGLE   =      491.5731  DIHED      =      672.9939
 VDWAALS =      291.3300  EEL     =        5.1582  HBOND      =        0.0000
 1-4 VDW =      231.1369  1-4 EEL =      -55.9856  RESTRAINT  =        0.0000


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1511       1.8420E+03     9.6606E-05     3.8660E-04     C138      305

 BOND    =      205.8039  ANGLE   =      491.5732  DIHED      =      672.9939
 VDWAALS =      291.3301  EEL     =        5.1582  HBOND      =        0.0000
 1-4 VDW =      231.1369  1-4 EEL =      -55.9856  RESTRAINT  =        0.0000

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|    Read coords time           0.00 ( 0.02% of Total)
|                Build the list             0.00 (91.00% of List )
|                Other                      0.00 ( 9.00% of List )
|             List time                  0.00 ( 0.06% of Nonbo)
|                   Short_ene time             2.49 (99.79% of Direc)
|                   Other                      0.01 ( 0.21% of Direc)
|                Direct Ewald time          2.50 (96.30% of Ewald)
|                Virial junk                0.00 ( 0.01% of Ewald)
|                Start synchronizatio       0.00 ( 0.01% of Ewald)
|                Other                      0.10 ( 3.68% of Ewald)
|             Ewald time                 2.59 (100.0% of Nonbo)
|             Other                      0.00 ( 0.01% of Nonbo)
|          Nonbond force              2.60 (73.30% of Force)
|          Bond/Angle/Dihedral        0.94 (26.66% of Force)
|          Other                      0.00 ( 0.03% of Force)
|       Force time                 3.54 (100.0% of Runmd)
|    Runmd Time                 3.54 (97.46% of Total)
|    Other                      0.09 ( 2.53% of Total)
| Total time                 3.63 (100.0% of ALL  )

| Number of list builds   :          1

| Highest rstack allocated:       1595
| Highest istack allocated:        319
|           Job began  at 14:36:19.970  on 06/17/2018
|           Setup done at 14:36:19.979  on 06/17/2018
|           Run   done at 14:36:23.605  on 06/17/2018
|     wallclock() was called   34788 times
